recipes/ruby.rb

```diff
@@ -18,10 +18,10 @@
 #
 
 node.default[:rbenv][:root_path] = '/opt/rbenv'
-node.default[:rbenv][:rubies] = [ '1.9.3-p484', '2.1.7', '2.3.0' ]
```

Comment:
I was on the fence on that, but wasn't sure about your opinnions on the these matters.